### ISSUE TITLE
not work in fragment

### ISSUE THREAD

**Issue by IbrahimSayed94** [stfalcon-studio/SmsVerifyCatcher#23]
may be found problem in permission

**Comment by akardas16** [comment]
yes I got this error Attempt to invoke virtual method 'void com.stfalcon.smsverifycatcher.SmsVerifyCatcher.onStart()' on a null object reference

**Comment by vishalpatel1327** [comment]
I also having same issue in fragment not working

**Comment by sepehr-alipour** [comment]
why it doesn't work in fragment?

**Comment by Alireza-hr** [comment]
Fixed !
you cant use inside Fragment becuse this function SmsVerifyCatcher() first parameter is Activity type 
you must definition a public function that setText code for Fragment from activity 
i get Object Fragment from FragmentManager and setText my code to Fragment inside Activity

**Comment by ezralazuardy** [comment]
another workaround is to use sharedViewModel with your parent activity
